docs/docs/sample-scripts/teams/list-teams-decluttering/sample.json

```diff
@@ -0,0 +1,53 @@
+[
+  {
+    "name": "lists-teams-decluttering",
+    "source": "pnp",
+    "title": "Lists information on decluttering teams",
+    "url": "https://pnp.github.io/cli-microsoft365/sample-scripts/teams/m365-teams-decluttering",
+    "creationDateTime": "2023-10-17",
+    "updateDateTime": "2023-10-17",
+    "shortDescription": "Lists all the teams to provide insights",
+    "longDescription": [
+      "Get insights about the teams in your tenant. Please keep in mind that you see only the teams that you have access to. For a full overview and a complete decluttering experience, the calling user has to be admin rights in SPO and Teams."
+    ],
+    "products": [
+      "Teams (SharePoint)"
```

Comment:
if I am not mistaken we should use one of the products that is mentioned in the [sample solution gallery](https://adoption.microsoft.com/en-us/sample-solution-gallery/?keyword=&sort-by=updateDateTime-true&page=1)

![image](https://github.com/pnp/cli-microsoft365/assets/58668583/e6a3ecd7-6293-4ee4-8c95-88f473b3787a)

otherwise it will not work properly for the filter

Reply:
Done!